### PR TITLE
feat: 934 my seedlot page cypress test

### LINKED ISSUE #934: My Seedlot page cypress test
Follow the test plan created in #920 
Create a my seedlot page cypress test. 

**Acceptance Criteria**
- [x] all functionalities are tested on my seedlots page

### PR DESCRIPTION
**Pull request by ngunner15**
# Description
Closes #934 

#### New
- My seedlot page test

#### Changed
- Create A class seedlot test
- Added more entries of rows by looping fixture data

#### Removed
- Intercepts from the tests

### How was this tested?
- [ ] 🧠 Not needed
- [x] 👀 Eyeball
- [x] 🤖 Added tests

###  What gif/image best describes this PR or how it makes you feel?

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-spar-40-frontend.apps.silver.devops.gov.bc.ca/)
- [Backend](https://nr-spar-1090-backend.apps.silver.devops.gov.bc.ca/actuator/health)
- [Oracle-API](https://nr-spar-1090-oracle-api.apps.silver.devops.gov.bc.ca/actuator/health)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-spar/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge.yml)

### CODE REVIEW

**Review by DerekRoberts — APPROVED** [review]
I want to approve based on that gif alone! But we'll have to wait until builds are passing. ;)

**Comment by craigyu** [comment]
`src/components/FavouriteActivities/index.tsx(72,21): error TS2322: Type '{ key: string; favObject: FavActivityType; index: number; }' `

Error is here, let's investigate together later

File: frontend/.eslintrc.json [hunk 1]
```diff
@@ -62,6 +62,7 @@
     "jsx-a11y/label-has-associated-control": [ 2, {
       "depth": 3
     }],
+    "semi":"warn",
```
[the remark this answers is not in the document]
Comment (ngunner15) [reply frontend/.eslintrc.json]: Old branch, was trying to fix eslinting

File: frontend/src/components/Card/FavouriteCard/index.tsx [hunk 2]
```diff
@@ -15,13 +15,11 @@ import SmallCard from '../SmallCard';
...
 interface FavouriteCardProps {
-  favObject: FavActivityType,
-  index: number
+  favObject: FavActivityType
 }
 
 const FavouriteCard = ({
-  favObject,
-  index
+  favObject
```
[the remark this answers is not in the document]
Comment (ngunner15) [reply frontend/src/components/Card/FavouriteCard/index.tsx]: Old branch, fixed it

**Review by RMCampos — APPROVED** [review]
LGTM! Great job @ngunner15 🎉

**Comment by craigyu** [comment]
Another PR is opened, closing this.

**Pull request closed without merging**